src/router/SignInScreen.tsx

```diff
@@ -1,9 +1,19 @@
 import { Button, Flex, Text } from '@chakra-ui/react'
 import { useConnectModal } from '@rainbow-me/rainbowkit'
 import { Trans } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { useEffect } from 'react'
 
 const SignInScreen = () => {
   const { openConnectModal } = useConnectModal()
+  const navigate = useNavigate()
+
+  // Redirect to the SAAS sign-in page
+  useEffect(() => {
+    if (!!import.meta.env.SAAS_URL) {
+      navigate('/signin')
+    }
+  }, [navigate])
 
```

Comment:
@selankon I do not comment on this because I undestand you will fix it in a future PR, right?

Reply:
Yes i am working on it on the https://github.com/vocdoni/ui-scaffold/pull/769 PR. But I have to understand first what exactly we want to do with this signin screen, there are some considerations to have. 

But, imo, lets talk this on the other PR when its ready.